templates/definition/meter/thor.yaml

```
@@ -0,0 +1,15 @@
+template: thor
+products:
+  - brand: my-PV
+    description:
+      generic: AC Thor
```

Man findet unterschiedliche Schreibweisen des Produktes im Netz. Der Hersteller schreibts aber immer so:
```suggestion
      generic: AC•THOR
```